examples/avian_physics/src/protocol.rs

```diff
@@ -43,12 +43,7 @@ impl PlayerBundle {
             id: PlayerId(id),
             position: Position(position),
             color: ColorComponent(color),
-            replicate: client::Replicate {
```

Comment:
Not needed since this is client->server. The replication_group is needed for server->client